### PR TITLE
Include all fix to a workable version for mooncake (Azure China)

### PR DESCRIPTION
**Pull request by yuanzhang9**
Changes on initiative definition and assignment due to missing build-in policy in mooncake:
Add new policy assignments à eslzArm\managementGroupTemplates\policyAssignments\china
Add new initiatives definition à src\resources\Microsoft.Authorization\policySetDefinitions
Update Bicep file on context loading à src\templates\initiatives.bicep, src\templates\policies.bicep
Update ARM template to load different assignments à eslzArm\eslzArm.json
 

Changes on role assignment due to service
Disable role “Security Operation” and “Network Management” as service “Microsoft.Support/*” is not available in mooncake. à src\templates\roles.bicep

Hide some option from Portal GUI as the service/policy/initiative are not available nor supported in mooncake. à eslzArm\eslz-portal.json
 

Change on service configuration:
Add new configuration file for Azure Defender for Cloud since the 2024-01-01version of API(Microsoft.Security/pricings) is not supported. à  eslzArm\subscriptionTemplates\mcmdfcConfiguration.json

### CODE REVIEW

**Review by Springstone — CHANGES_REQUESTED** [review]
Couple of things to review, and please address the conflicts

File: src/resources/Microsoft.Authorization/policySetDefinitions/Enforce-ALZ-Decomm.json [hunk 1]
```diff
@@ -13,8 +13,7 @@
         "source": "https://github.com/Azure/Enterprise-Scale/",
         "alzCloudEnvironments": [ 
           "AzureCloud",
-          "AzureChinaCloud",
-          "AzureUSGovernment"
+          "AzureChinaCloud"
```
[the remark this answers is not in the document]
Comment (yuanzhang9) [reply src/resources/Microsoft.Authorization/policySetDefinitions/Enforce-ALZ-Decomm.json:16]: Yes, nice spot. It should be "AzureUSGovernment"  instead.

File: src/resources/Microsoft.Authorization/policySetDefinitions/Deploy-Private-DNS-Zones.AzureChinaCloud.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "displayName": "Configure Azure PaaS services to use private DNS zones",
     "description": "This policy initiative is a group of policies that ensures private endpoints to Azure PaaS services are integrated with Azure Private DNS zones",
     "metadata": {
-      "version": "1.0.1",
+      "version": "1.2.0",
```
[the remark this answers is not in the document]
Comment (yuanzhang9) [reply src/resources/Microsoft.Authorization/policySetDefinitions/Deploy-Private-DNS-Zones.AzureChinaCloud.json]: I've adjusted it to 1.1.0 and completed the test. Thanks~

**Comment by yuanzhang9** [comment]
One more update for "Routing intent of vWAN hub", included in this PR as well.

**Review by Springstone — APPROVED** [review]

**Pull request merged**